src/Symfony/Component/Console/Application.php

```diff
@@ -103,7 +101,8 @@ public function setDispatcher(EventDispatcherInterface $dispatcher)
      *
      * @return int 0 if everything went fine, or an error code
      *
-     * @throws \Exception When doRun returns Exception
+     * @throws \Exception When doRun threw an exception and catchExceptions is false
+     * @throws \Throwable When doRun threw a throwable
```

Comment:
this should be removed entirely actually. It does not make any sense to document that it throws an exception when the inner code threw an exception. PHP does not require each class to document exceptions thrown in called codes (unlike Java)